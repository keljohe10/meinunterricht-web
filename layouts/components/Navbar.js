import { Box, Flex } from '@chakra-ui/react';

export default function Nav() {
  return (
    <>
      <Box bg='gray.900' px={4}>
        <Flex h={16} alignItems={'center'} justifyContent={'space-between'}>
          <Box color='white'>MeinUnterricht</Box>
        </Flex>
      </Box>
    </>
  );
}
