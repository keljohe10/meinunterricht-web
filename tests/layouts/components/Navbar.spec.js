/**
 * @jest-environment jsdom
 */

import React from 'react';
import '@testing-library/jest-dom/extend-expect';

import { render, cleanup } from '@/tests/testUtils';

import Navbar from '@/layouts/components/Navbar';

beforeEach(cleanup);

describe('Navbar', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = render(<Navbar />);
  });

  it('matches snapshot', () => {
    expect(wrapper.asFragment()).toMatchSnapshot();
  });

  it('should render the logo', () => {
    expect(
      wrapper.getByText('MeinUnterricht', { exact: false }),
    ).toBeInTheDocument();
  });
});
