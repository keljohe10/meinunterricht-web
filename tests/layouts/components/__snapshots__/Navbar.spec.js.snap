// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-fl07t2"
  >
    <div
      class="css-jfo7z"
    >
      <div
        class="css-kbn7if"
      >
        MeinUnterricht
      </div>
    </div>
  </div>
</DocumentFragment>
`;
